governance.md

```diff
@@ -32,33 +32,65 @@ Every code owner is expected to:
 
 To suggest a change in the ownership of a given repository part, create a PR with the required changes in the `CODEOWNERS` file in the project's repository. The required number of code owners needs to approve the PR for the changes to take place. Read [here](https://github.com/kyma-project/community/blob/master/guidelines/internal-guidelines/repository-template/template/CODEOWNERS) how to set up and modify owners of the given repository folders and files.
 
-## The decisions-making process
+## Decision making
 
-In general, the relevant Special Interest Groups (SIGs) and Working Groups (WGs) make the decisions that affect the project, including its structure, functionalities, components, or work of the project teams. However, the organizational decisions and those that relate to the product strategy belong to the Kyma Council.
+In general, the Special Interest Groups (SIGs) and Working Groups (WGs) make decisions that affect the project, including its structure, functionalities, components, or work of the project teams. However, the organizational decisions and those that relate to the product strategy are made by the Kyma Council.
 
-SIGs and WGs follow the "lazy consensus" decision-making process which assumes that:
+SIGs and WGs follow the **lazy consensus** approach to decision making which assumes that:
 
 * All SIG/WG members have an equal voice in the decision-making process.
 * Silence is consent. By default, lack of objections to a proposed decision means a silent approval.
 * Any objections are good opportunities for healthy and constructive discussions.
 
 > **NOTE:** The described approach only concerns the decisions made by SIGs and WGs. It does not affect any Kyma decisions made during daily team activities.
 
-### Process
+At the [Kyma project decisions board](https://github.com/kyma-project/community/issues/106#boards?labels=decision&activeFilters=labels&repos=139590666,151691065,139590577,139590641,139590616,139590701,147495537,139847219) you may see the current state of processed decisions in the Kyma project.
 
-The SIG/WG decison-making process looks as follows:
+The SIG/WG decison-making process is depicted as follows:
 
 ![Decision-making process](assets/decision-making-process.png)
 
```

Comment:
shouldn't we attach here the new diagram?

Reply:
@agolawska yes it will be attached automatically, once that PR is merged.